libs/core/src/lib/tabs/tabs.module.ts

```diff
@@ -29,7 +45,16 @@ import { TabItemDirective } from './tab-item/tab-item.directive';
         TabLoadTitleDirective,
         TabNavComponent,
         TabItemDirective,
-        TabLinkDirective
+        TabLinkDirective,
+        TabTagDirective,
+        TabIconDirective,
+        TabCountDirective,
+        TabLabelDirective,
+        TabProcessDirective,
+        TabHeaderDirective,
+        TabCounterHeaderDirective,
+        TabProcessIconDirective,
+        TabSeparator
```

Comment:
These should also be exported in the `api-files.ts` fille

Reply:
There are exported all directives from `tab-directives` in public_api.ts